modules/oe_editorial_corporate_workflow/tests/src/Traits/CorporateWorkflowTrait.php

```diff
@@ -42,23 +43,38 @@ protected function moderateNode(NodeInterface $node, string $target_state): Node
       'published',
     ];
 
-    $current_state = $node->get('moderation_state')->value;
+    $current_state = $entity->get('moderation_state')->value;
     if ($current_state === $target_state) {
-      return $node;
+      return $entity;
     }
 
     $pos = array_search($current_state, $states);
     foreach (array_slice($states, $pos + 1) as $new_state) {
-      $node = $revision ?? $node;
-      $revision = $this->getEntityTypeManager()->getStorage('node')->createRevision($node);
+      $entity = $revision ?? $entity;
+      $revision = $this->getEntityTypeManager()->getStorage('node')->createRevision($entity);
```

Comment:
The getStorage() should be based on the entity type. This won't work for other entities.